### PR TITLE
Angular animations

### PR DESCRIPTION
**Pull request by andii0890**
I was working on this branch, presenting the animation section: a donut chart and a scroll mouse, the latter instead of the button I had before, I liked it more and is more direct about what follows.

### CODE REVIEW

File: my-page/src/app/app.component.html [hunk 1]
```diff
@@ -15,7 +15,10 @@
 	<mat-grid-tile>
 		<app-contact [texts]="texts"></app-contact>
 	</mat-grid-tile> 
-  </mat-grid-list>  
+  </mat-grid-list>   -->
```
Comment (LuisRBarreras) [inline my-page/src/app/app.component.html:18]: Check --> is necessary?

File: my-page/src/app/donut-chart/donut-chart.component.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div>
+	<div style="display: block"></div>
```
Comment (LuisRBarreras) [inline my-page/src/app/donut-chart/donut-chart.component.html:2]: move style to a css file

File: my-page/src/app/app.component.html (hunk 1 above)
Comment (andii0890) [reply my-page/src/app/app.component.html]: nope, it's already deleted.

File: my-page/src/app/donut-chart/donut-chart.component.html (hunk 2 above)
Comment (andii0890) [reply my-page/src/app/donut-chart/donut-chart.component.html]: ready, it's where it should go

**Pull request merged**